clang/lib/AST/ExprConstant.cpp

```diff
@@ -15314,6 +15314,32 @@ bool FloatExprEvaluator::VisitCallExpr(const CallExpr *E) {
       Result = RHS;
     return true;
   }
+
+  case Builtin::BI__builtin_fmaximum_num:
+  case Builtin::BI__builtin_fmaximum_numf:
+  case Builtin::BI__builtin_fmaximum_numl:
+  case Builtin::BI__builtin_fmaximum_numf16:
+  case Builtin::BI__builtin_fmaximum_numf128: {
+    APFloat RHS(0.);
+    if (!EvaluateFloat(E->getArg(0), Result, Info) ||
```

Comment:
Missing constexpr evaluation tests

Reply:
Thanks.
@tbaederr It seems that `__builtin_fmaximum_num` cannot work with `-fexperimental-new-constant-interpreter`.
I guess I must miss something.
Any idea?

Reply:
Well you didn't implement that. If you add a new file anyway, you don't have to add a run line for it.